cluster/storage/src/main/java/com/linkedin/openhouse/cluster/storage/adls/ADLSStorage.java

```diff
@@ -0,0 +1,32 @@
+package com.linkedin.openhouse.cluster.storage.abs;
+
+import com.linkedin.openhouse.cluster.storage.BaseStorage;
+import com.linkedin.openhouse.cluster.storage.StorageClient;
+import com.linkedin.openhouse.cluster.storage.StorageType;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Lazy;
+import org.springframework.stereotype.Component;
+
+/**
+ * The ADLS Storage class is an implementation of the Storage interface for cloud storage with Azure
+ * Data Lake Storage.
+ */
+@Component
+public class ADLSStorage extends BaseStorage {
+
+  @Autowired @Lazy
+  private ADLSStorageClient
+      adlsStorageClient; // declare client class to interact with ADLS filesystem
```

Comment:
Can you please be consistent with the comment placement? The rest of code places comment on the line before the function.